Corvus.Extensions.Samples/StringExtensionsSample.dib

```diff
@@ -0,0 +1,181 @@
+#!markdown
+
+## StringExtensions - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+#!markdown
+
+Add using statement for `Corvus.Extensions` namespace
+
+#!csharp
+
+using System.IO;
+using Corvus.Extensions;
+
+#!markdown
+
+### `AsBase64()`
+
+#!markdown
+
+Convert the provided string to a base 64 representation of its byte representation in a particular encoding.
+
+#!markdown
+
+#### Examples
+
+The example below converts a string's byte representation in Unicode to a base 64 representation.
+
+#!csharp
+
+string hello = "Hello";
+return hello.AsBase64(Encoding.ASCII);	string encoded = hello.AsBase64(Encoding.Unicode);
+
+#!markdown
+
+### `Base64UrlEncode()`
+
+#!markdown
+
+Convert the provided string to a base 64 representation of its byte representation in the UTF8 encoding,
+with a URL-safe representation.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string url = "https://www.youtube.com/watch?v=pcc-EXzK3vI";
+string urlEncoded = url.Base64UrlEncode();
+    urlEncoded
+
+#!markdown
+
+### `Base64UrlDecode()`
+
+#!markdown
+
+Convert the provided string from a base 64 representation of its byte representation in the UTF8 encoding
+with a URL-safe representation.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string urlEncoded = "aHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g_dj1wY2MtRVh6SzN2SQ";
+string urlDecoded = urlEncoded.Base64UrlDecode();
+urlDecoded
+
+#!markdown
+
+### `AsStream()`
+
+Provide a stream over the string in the specified encoding.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string myString = "hello";
+Stream myStream = myString.AsStream(Encoding.ASCII);
+return myStream.ToString();
+
+#!markdown
+
+### `EscapeContentType()`
+
+#!markdown
+
+Escape a content type string.
+
+#!csharp
+
+string url = "https://www.youtube.com/watch?v=pcc-EXzK3vI";
+return url.EscapeContentType();
+
+#!markdown
+
+### `FromBase64()`
+
+#!markdown
+
+Decode a string from a base64-encoded byte array with the specified text encoding.
+
+#!csharp
+
+string base64String = "SGVsbG8=";
+return base64String.FromBase64();
+
+#!markdown
+
+### `GetGraphemeClusters()` 
+
+#!markdown
+
+Enumerate the grapheme clusters in a string.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string myString = "hello";
+return myString.GetGraphemeClusters();
+
+#!markdown
+
+### `Reverse()`
+
+Reverse the string.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string myString = "hello";
+return myString.Reverse();
+
+#!markdown
+
+### `UnescapeContentType()`
+
+Unescape a content type string.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string myString = "https%3A%60%60www%7Cyoutube%7Ccom%60watch%3Fv%3Dpcc-EXzK3vI";
+return myString.UnescapeContentType();
```

Comment:
Same comment - this isn't a content type string

Reply:
Fixed ✅